src/index.ts

```diff
@@ -28,6 +31,41 @@ export const execute = async () => {
       executed = true;
     });
 
+  commander
+    .command("alias [pair]")
+    .description("Upsert an alias with <name>=[issueId]")
+    .action(async (alias: string) => {
+      executed = true;
+
+      const printAliases = (aliases: IAliases | null) =>
+        aliases
+          ? Object.entries(
+              aliases
+            ).forEach(([name, { ticketId, description }]) =>
+              Logger.info(
+                `${name}=${ticketId}${description ? `,${description}` : ""}`
```

Comment:
Could you put a ' ' after the ',' ?

Reply:
With this format (without space) the output is consistent with the expected input. Is this intended behavior?